core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -545,21 +545,25 @@ public function getObjectsInContainer($path)
 
 
     /**
-     * @param $path
+     * @param string $path
+     * @param bool $isNew
      *
      * @return array|bool
      */
-    public function getMetaData($path)
+    public function getMetaData($path, $isNew = false)
     {
         try {
             $data['path'] = $path;
             $data['timestamp'] = $this->filesystem->lastModified($path);
             $data['size'] = $this->filesystem->fileSize($path);
             $data['mimetype'] = $this->filesystem->mimeType($path);
             $data['type'] = $data['mimetype'] === 'directory' ? 'dir' : 'file';
-
         } catch (FilesystemException | UnableToRetrieveMetadata $e) {
-            $this->xpdo->log(modX::LOG_LEVEL_ERROR, $e->getMessage());
+            if (!$isNew) {
+                $this->xpdo->log(modX::LOG_LEVEL_ERROR, $e->getMessage());
+            } else {
+                $this->xpdo->log(modX::LOG_LEVEL_ERROR, $this->xpdo->lexicon('file_info_metadata_new', ['file' => $path]));
```

Comment:
Isn't the message we log debug information instead of an error?

Reply:
It is, yes. A couple things here: I thought it might be helpful for it to be noticable for testing. I did have in mind knocking it down to INFO in the end. Also, I see this disappearing at some point, hopefully soon. I looked very carefully at how the file handling process works now and find it rather convoluted. Why is getMetaData _any_ part of checking whether a file exists or not? This method should strictly be used for what it implies its use is: Fetching the metadata of an existing file.